core-bundle/contao/themes/flexible/styles/main.css

```
@@ -2977,42 +2977,34 @@ fieldset.collapsed > legend:not(:has(> button)) {
 /* Tablet */
 @media (max-width: 991px) {
 	#container {
-		display: block;
-	}
-
-	#main, #left {
-		float: none;
+		overflow: clip;
```

```suggestion
		overflow-x: clip;
```
Just a nit, feel free to ignore